src/library/scala/collection/StringOps.scala

```diff
@@ -1057,19 +1057,18 @@ final class StringOps(private val s: String) extends AnyVal {
     true
   }
 
-  /** Applies a binary operator to a start value and all chars of this string,
-    * going left to right.
-    *
-    *  @param   z    the start value.
-    *  @param   op   the binary operator.
-    *  @tparam  B    the result type of the binary operator.
-    *  @return  the result of inserting `op` between consecutive chars of this string,
-    *           going left to right with the start value `z` on the left:
-    *           {{{
-    *             op(...op(z, x_1), x_2, ..., x_n)
-    *           }}}
-    *           where `x,,1,,, ..., x,,n,,` are the chars of this string.
-    *           Returns `z` if this string is empty.
+  /** Applies the given binary operator to an initial value and all chars in this
+    * string, going left to right. Returns the initial value if this string is
+    * empty.
+    *
+    * If `x,,1,,`, `x,,2,,`, ..., `x,,n,,` are the chars in this string, the
+    * result is `op( op( ... op( op(z, x,,1,,), x,,2,,) ... ), x,,n,,)`.
+    *
+    *  @param    z       An initial value.
+    *  @param    op      A binary operator.
+    *  @tparam   B       The result type of the binary operator.
+    *  @return           The result of applying `op` to `z` and all chars in this string,
+    *                    going left to right. Returns `z` if this string is empty.
```

Comment:
No one else will suggest it, but for the record, this is an opportunity to correct misaligned asterisks. I don't know if the rule is that the first person to add doc sets the style or the last. I'm pretty sure that whoever publishes 2.14 has the last say, until Dotty takes over.  I wonder if Dotty would accept a PR for "starless" doc comments, which begin with a line comment in column 1 (that is not a `//>` directive).

Reply:
Regardless of the rule, this file has a mix of at least two styles with no discernible pattern. I will fix the alignment for the docstrings I touched.